util-bukkit/src/main/i18n/templates/strings.properties

```diff
@@ -1045,415 +1033,385 @@ observer.tools.flyspeed.hyperspeed = Hyperspeed
+match.join.success.team = You joined {0}
 
 # {0} = a team name
-team.balanceWarning = Teams will be auto-balanced if {0} remains stacked
+match.broadcasts.balanceWarning = Teams will be auto-balanced if {0} remains stacked
 
 # {0} an amount of time
-huddle.instructions = Your team now has {0} to strategize before the match starts
+match.huddle.start = Your team now has {0} to strategize before the match starts
 
 # {0} = The current match number
-command.match.matchInfo.title = Match #{0}
-command.match.matchInfo.time = Time
-command.match.matchInfo.matchTime = Match Time
-command.match.matchInfo.goals = Goals
-command.match.matchInfo.players = Players
-command.match.matchInfo.observers = Observers
-command.match.matchInfo.ranking = Ranking
-command.match.matchInfo.ranking.message = The result of this match will affect your win/loss stats and ranking!
+match.matchInfo.title = Match #{0}
+match.matchInfo.time = Time
+match.matchInfo.matchTime = Match Time
+match.matchInfo.goals = Goals
+match.matchInfo.players = Players
+match.matchInfo.observers = Observers
+match.matchInfo.ranking.header = Ranking
+match.matchInfo.ranking.message = The result of this match will affect your win/loss stats and ranking!
 
 # {0} = user being forced
 # {1} = team being forced onto
 # {2} = last team before force
-command.team.force.success = {0} forced onto {1} from {2}
-
-command.team.shuffle.success = Teams successfully shuffled.
-
+match.team.force.success = {0} forced onto {1} from {2}
+match.team.shuffle.success = Teams successfully shuffled.
 # {0} = old name
 # {1} = new name
-command.team.alias.success = {0} successfully renamed to {1}
+match.team.alias.success = {0} successfully renamed to {1}
 
 # {0} = team name
-command.gameplay.myteam.message = You are on {0}
+match.myTeam.yourTeam = You are on {0}
 
 # {0} = Maximum build height (number)
-match.maxBuildHeightWarning = You have reached the maximum build height ({0} blocks)
+match.playableArea.maxBuildHeightWarning = You have reached the maximum build height ({0} blocks)
 
-match.lane.exit = You have left your lane
+match.laneExit = You have left your lane
 
-match.scoreboard.default.title = Match
-match.scoreboard.objectives.title = Objectives
-match.scoreboard.scores.title = Scores
-match.scoreboard.blitz.title = Blitz
-match.scoreboard.livesRemaining.title = Lives Remaining
-match.scoreboard.playersRemaining.title = Players Remaining
-match.scoreboard.rage.title = Blitz: RAGE
+match.scoreboard.title.default = Match
+match.scoreboard.title.objectives = Objectives
+match.scoreboard.titles.scores = Scores
+match.scoreboard.title.blitz = Blitz
+match.scoreboard.title.livesRemaining = Lives Remaining
+match.scoreboard.title.playersRemaining = Players Remaining
+match.scoreboard.title.blitzRage = Blitz: RAGE
 
-# {0} = "1"
-countdown.singularCompound = {0} second
+match.join.denied.teamsFull = Teams are full
+match.join.denied.capacity = Teams are at capacity
+match.join.denied.matchFull = Match is full
```

Comment:
There is a lot of strings involving `join`, how about we make that it's own category?

Reply:
Agreed, there is a lot of them...

Reply:
We can put them with join.